EIPS/eip-1.md

```diff
@@ -67,54 +87,112 @@ These calls generally result in a "rough consensus" around what EIPs should be i
 
 *In short, your role as the champion is to write the EIP using the style and format described below, shepherd the discussions in the appropriate forums, and build community consensus around the idea.*
 
+
 ### EIP Process
 
 The following is the standardization process for all EIPs in all tracks:
 
 ![EIP Status Diagram](../assets/eip-1/EIP-process-update.jpg)
 
-**Idea** - An idea that is pre-draft. This is not tracked within the EIP Repository.
+#### Idea
 
-**Draft** - The first formally tracked stage of an EIP in development. An EIP is merged by an EIP Editor into the EIP repository when properly formatted.
+An idea that is pre-draft. This is not tracked within the EIP Repository.
 
-**Review** - An EIP Author marks an EIP as ready for and requesting Peer Review.
+#### Draft
 
-**Last Call** - This is the final review window for an EIP before moving to `Final`. An EIP editor will assign `Last Call` status and set a review end date (`last-call-deadline`), typically 14 days later.
+The first formally tracked stage of an EIP in development. An EIP is merged by an EIP Editor into the EIP repository when it meets some initial requirements: It should be technically sound, the Title and Preamble must be correct, and all included Sections should be readable.  Problems with language like spelling, grammar, markup, etc. are only considered warnings.
 
-If this period results in necessary normative changes it will revert the EIP to `Review`.
+#### Review
 
-**Final** - This EIP represents the final standard. A Final EIP exists in a state of finality and should only be updated to correct errata and add non-normative clarifications.
+An EIP Author marks an EIP as ready for and requesting Peer Review.  An Editor will merge the EIP when it meets all EIP requirements: it should be a technically sound, complete specification, properly formatted, with correct language, markup and external references.
```

Comment:
```suggestion
An EIP Author marks an EIP as ready for and requesting Peer Review.  An Editor will merge the status change when the proposal meets all EIP requirements: it should be a complete specification, properly formatted, with correct language, markup and external references.
```

Reply:
Still wanting to keep "technically sound".

Reply:
I'm not qualified to judge technical soundness for all EIPs. Maybe we go with a lower bar of plausibly sound?

Reply:
Yeah, Technical Assistants could help editors feel qualified/informed enough to make this assessment, but delegating that assesment to the consensus of a WG (however tightly or loosely defined) of, say, 2 or more competing implementers and 1 or more editors-or-technical-assistants sounds like a great option if it's on the table!

Reply:
I'm resolving this unchanged for now.  Outside review is helpful and often necessary, but in the end the Editors have to make the call.